rich/live.py

```diff
@@ -59,7 +71,9 @@ def __init__(
         transient: bool = False,
         redirect_stdout: bool = True,
         redirect_stderr: bool = True,
-        vertical_overflow: VerticalOverflowMethod = "ellipsis",
+        vertical_overflow: Union[
```

Comment:
I'm not keen on overloading a parameter like this. If it becomes a general purpose extension mechanism, it could quickly become unwieldy. I'd prefer to keep the message a separate value.